services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -306,7 +310,9 @@ async def stop_computation_project(
         return ComputationTaskOut(
             id=project_id,
             state=pipeline_state,
-            pipeline_details=await compute_pipeline_details(complete_dag, pipeline_dag),
+            pipeline_details=await compute_pipeline_details(
+                complete_dag, pipeline_dag, tasks
```

Comment:
third argumetn is tasks or ``all_comp_tasks``??

Reply:
aha ... it is another function (sorry in the review mode it was collapsed). 
MINOR: perhaps use the same variable names in all related functions?

Reply:
ok discussed